Doc/howto/descriptor.rst

```diff
@@ -870,6 +870,8 @@ care of lookups or updates:
             conn.execute(self.store, [value, obj.key])
             conn.commit()
 
+*Formatting string is not recommended for building SQL queries. Since it is just an example and we didn't want to make the code more complicated (in fact, using them in our example is quite justified), we decided to use it. You have to validate the data before putting it into a query string or trust an end-user, which sometimes is not the best option. Use placeholders instead, as we did in ``WHERE`` clause.
```

Comment:
```suggestion
.. note::
    Formatting string is not recommended for building SQL queries.
    Since it is just an example and we didn't want to make the code
    more complicated (in fact, using them in our example is quite justified),
    we decided to use it. You have to validate the data before putting it
    into a query string or trust an end-user, which sometimes is not the best
    option. Use placeholders instead, as we did in ``WHERE`` clause.

```

Reply:
For future reference: .rst files are indented with three spaces, not four. You'll see that by using three spaces, the text will line up nicely with the `note::` directive.